From c668a56d8dd561e1c5ce1780c3ad52fdcea13c9f Mon Sep 17 00:00:00 2001
From: Mike Zeller <mike.zeller@joyent.com>
Date: Wed, 16 May 2018 21:39:39 +0000
Subject: [PATCH] TRITON-348 breaks setting up fabrics when
 sdc_underlay_assignment is set to manual Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 lib/post-setup/fabrics.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/post-setup/fabrics.js b/lib/post-setup/fabrics.js
index 86ffa77..6af80f1 100644
--- a/lib/post-setup/fabrics.js
+++ b/lib/post-setup/fabrics.js
@@ -631,6 +631,7 @@ Fabrics.prototype.checkAssignment = function checkAssignment(cb) {
             return;
         }
         cb(null);
+        return;
     }
 
     self.sdcadm.napi.getNetworkPool(self.data.sdc_underlay_pool,
-- 
2.21.0

